<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="income">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/income.png</normaloff>:/images/income.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inValue">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>10000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/outcome.png</normaloff>:/images/outcome.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outValue">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>800</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="leftBottomLayout">
          <item>
           <widget class="QPushButton" name="comment">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>comment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="commit">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>commit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addAccount">
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>income</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>outcome</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="rightMidLayout">
            <item>
             <widget class="QToolButton" name="today_detail">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/images/arrow.png</normaloff>:/images/arrow.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>This Week</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>This Month</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>This Season</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>This Year</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Export Diagram</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="font">
                 <font>
                  <family>Comic Sans MS</family>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Export List</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_3"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Debt Record</string>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/images/debt.png</normaloff>:/images/debt.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_3"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder></zorder>
   <zorder>today_detail</zorder>
   <zorder>comboBox</zorder>
   <zorder>horizontalLayoutWidget_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Label</string>
    </property>
    <addaction name="actionZengjia"/>
    <addaction name="actionDelete_label"/>
    <addaction name="actionResort_Label"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>set alart</string>
    </property>
    <addaction name="actionShezhi"/>
    <addaction name="actionRemove_threshold"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Accouting"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionZengjia">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/label.png</normaloff>:/images/label.png</iconset>
   </property>
   <property name="text">
    <string>add label</string>
   </property>
  </action>
  <action name="actionShezhi">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/threshold.png</normaloff>:/images/threshold.png</iconset>
   </property>
   <property name="text">
    <string>threshold seting</string>
   </property>
  </action>
  <action name="actionDelete_label">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/delete_label.png</normaloff>:/images/delete_label.png</iconset>
   </property>
   <property name="text">
    <string>delete label</string>
   </property>
  </action>
  <action name="actionResort_Label">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/resort_label.png</normaloff>:/images/resort_label.png</iconset>
   </property>
   <property name="text">
    <string>Resort Label</string>
   </property>
  </action>
  <action name="actionRemove_threshold">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>remove threshold</string>
   </property>
  </action>
  <action name="actionUpload_Now">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload Now</string>
   </property>
  </action>
  <action name="actionCancel_Uplaod">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancel Upload</string>
   </property>
  </action>
  <action name="actionAbout_Accouting">
   <property name="text">
    <string>About Accouting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
